migrations/move_site_icons.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ *
+ * phpBB Browser Push Notifications. An extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2024, phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+namespace phpbb\webpushnotifications\migrations;
+
+use phpbb\db\migration\container_aware_migration;
+use phpbb\filesystem\exception\filesystem_exception;
+use phpbb\filesystem\filesystem;
+
+class move_site_icons extends container_aware_migration
+{
+	private const NEW_ICON_DIR = 'images/site_icons';
+	private const OLD_ICON_DIR = 'images/icons';
+
+	/* @var filesystem $filesystem */
+	private $filesystem;
+
+	public function effectively_installed()
+	{
+		return $this->get_filesystem()->exists($this->container->getParameter('core.root_path') . self::NEW_ICON_DIR);
+	}
+
+	public static function depends_on()
+	{
+		return ['\phpbb\webpushnotifications\migrations\add_acp_pwa_configs'];
+	}
+
+	public function update_data(): array
```

Comment:
Add `revert_data()`? Basically removing the icons dir.

Reply:
No we leave the directory behind. It could be used for other images, like favicons and additional touch icons. We never want to delete any files outside of the extension.